testing_and_setup/compass/ocean/global_ocean/template_forward.xml

```diff
@@ -3,8 +3,6 @@
 		<option name="config_ocean_run_mode">'forward'</option>
 		<option name="config_block_decomp_file_prefix">'graph.info.part.'</option>
 		<option name="config_time_integrator">'split_explicit'</option>
-		<option name="config_use_mom_del2">.true.</option>
-		<option name="config_use_mom_del4">.true.</option>
```

Comment:
Do these default true?  Not sure why they are being removed.  Do we really want the values to be defaulted to true?  Each simulation will require someone to consider this -- at small scale `del2` should be used and `del4` at large scale, for instance.  Having both default `.false` ensures this level of thought, but perhaps having both on does no harm in general.

Reply:
Thanks. I changed this back.